brayns/common/Transform.cpp

```diff
@@ -56,11 +56,29 @@ const Vector3f &Transform::getScale() const noexcept
 
 void Transform::setScale(const Vector3f &value) noexcept
 {
-    _updateValue(_scale, value);
+    _scale = value;
 }
 
 Matrix4f Transform::toMatrix() const
 {
     return glm::translate(Matrix4f(1.), _translation) * glm::mat4_cast(_rotation) * glm::scale(Matrix4f(1.), _scale);
 }
+
+bool operator==(const Transform &lhs, const Transform &rhs)
+{
+    auto &lhsT = lhs.getTranslation();
+    auto &lhsR = lhs.getRotation();
+    auto &lhsS = lhs.getScale();
+
+    auto &rhsT = rhs.getTranslation();
+    auto &rhsR = rhs.getRotation();
+    auto &rhsS = rhs.getScale();
+
+    return lhsT == rhsT && lhsR == rhsR && lhsS == rhsS;
+}
+
+bool operator!=(const Transform &lhs, const Transform &rhs)
+{
+    return !(lhs == rhs);
+}
```

Comment:
I would put that inside the Transform class and use rhs._translation directly.

For that we can also use std::tie(_translation, rotation, scal) == std::tie(other._translation, other.rotation, other.scale).